flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/utils/StreamingWithStateTestBase.scala

```diff
@@ -63,7 +64,7 @@ class StreamingWithStateTestBase(state: StateBackendMode) extends StreamingTestB
   override def before(): Unit = {
     super.before()
     // set state backend
-    baseCheckpointPath = tempFolder.toFile
+    baseCheckpointPath = Files.createTempDirectory("junit").toFile
```

Comment:
```suggestion
    baseCheckpointPath = Files.createTempDirectory(tempFolder, "junit").toFile
```
maybe, we should still utilize the temporary folder that is created in the parent class to utilize JUnit5's temporary folder feature.